### DISCUSSION TITLE
Setup for both docker and local

### DISCUSSION THREAD

**Discussion by BFRS01912** [phpro/grumphp#1044] [Q&A]
I need to check the syntax errors in my code while committing, I did the setup for grumphp with PHP parser. 

I have a local setup and a docker setup for the project repository and both share the same .git folder, by default the working directory for both setup is different. 

The problem I'm facing is when I'm committing my changes from a local setup it is giving me a pre-commit error ".git/hooks/pre-commit: 12: exec: /var/www/.../vendor/bin/grumphp: not found" because the pre-commit file has docker working file directory. And when I'm updating the path directory with a local one, the same error occurs for the docker setup one.

The contents of my grumphp.yml file are
parameters:
    tasks:
        phpparser:
            ignore_patterns: []
            kind: php7
            visitors: {}
            triggered_by: [PHP]

Please help me by telling me a way to use grumphp for both setups and I don't have to update the git hooks pre-commit file again and again.

**Comment by veewee** [comment]
Hello,

I'm a bit confused about your question.
Are you committing inside the docker container then?

In any case: the easiest way is to change the exec command to run the checks inside the docker container.
You commit locally, but grumphp will be ran on the docker container.

It's documented here 
https://github.com/phpro/grumphp/blob/master/doc/parameters.md
